playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -64,6 +64,14 @@
   - name: deconfigure yum repositories
     import_tasks: tasks/remove_yum.yml
 
+  - name: add get-node-logs script
+    copy:
+      src: files/get-node-logs
+      dest: /usr/local/bin/get-node-logs
+      owner: root
+      group: root
+      mode: 0550
```

Comment:
0755 would be fine, wouldn't it?

Reply:
only root can run it so is there any point exposing it to others? 0750?

Reply:
0755 please.